WalletWasabi.Fluent/Controls/QuestionControl.axaml.cs

```diff
@@ -0,0 +1,63 @@
+using System.Windows.Input;
+using Avalonia;
+using Avalonia.Controls;
+using Avalonia.Media;
+
+namespace WalletWasabi.Fluent.Controls
+{
+	public enum HighlightedButton
+	{
+		None,
+		YesButton,
+		NoButton,
+		Both
+	}
+
+	public class QuestionControl : Label
```

Comment:
wonder if contentcontrol or just templated control is more appropriate?

Reply:
Good point, fixed